service/src/test/resources/ActivityFinishedEventDuplicatedKeys.json

```diff
@@ -12,22 +12,14 @@
         "host": "host",
         "name": "name",
         "uri": "http:\/\/java.sun.com\/j2se\/1.3\/",
-        "serializer": {
-          "groupId": "G",
-          "artifactId": "A",
-          "version": "V"
-        },
-        "serializer": {
-          "groupId": "G",
-          "artifactId": "A",
-          "version": "V"
-        }
+        "serializer":"pkg://mvn",
+        "serializer":"pkg://mvn"
```

Comment:
2 times? and please complete the serializer

Reply:
yes, you can see the file name, we are checking for duplicateKeys to see if it throws error.